pipelines/h6c/rtp/v2/h6c_rtp_stage_1.toml

```diff
@@ -436,6 +437,16 @@ args = ["{basename}",
         "${NOTEBOOK_OPTS:nb_output_repo}",
         "${NOTEBOOK_OPTS:git_push}",
         ]
+[ALL_SKY_IMAGE_SNAPSHOT]
+prereqs = ["CALIBRATE_FILE_NOTEBOOK"]
+mem = 1200
```

Comment:
I highly doubt this will run in this amount of memory. Are you sure you didn't mean 12000?

(Generally an open pyuvdata file is a lot larger in memory than on disk)

Reply:
what are the units? Thought it was megabytes.  Went back and dug through more sacct. Highest RAM usage I can find is 6672104K (6.7GB).

Reply:
1200 is ~1.2 GB

Reply:
I will increase to 7000.